test/fancylogger.py

```diff
@@ -661,27 +630,27 @@ class ScreenLogFormatterFactoryTest(TestCase):
 
     def test_colorize_never(self):
```

Comment:
why do we keep the colorize tests? they are not supposed to do anything anymore, right?